src/go/log/mock_log/mock_logger.go

```diff
@@ -0,0 +1,145 @@
+// Code generated by MockGen. DO NOT EDIT.
+// Source: github.com/magma/magma/log (interfaces: Logger)
+
+// Package mock_log is a generated GoMock package.
+package mock_log
+
+import (
+	reflect "reflect"
+
+	gomock "github.com/golang/mock/gomock"
+	log "github.com/magma/magma/log"
```

Comment:
go question..
```suggestion
    "reflect"

	"github.com/golang/mock/gomock"
	"github.com/magma/magma/log"
```
does that work too? or are we doing something special here?

Reply:
your suggestion definitely works and would be preferred, but this is codegen'd output from `gomock` (explicit package names can be seen in generated pb.go files as well). I didn't bother fixing up any generated code since it would require updating the gomock/protoc-gen-go tooling to make it stable.

hopefully files like this should be collapsed/hidden by default in the GH UI.

Reply:
oh! I see. It makes sense if it's generated. Weird that GitHub didn't detect this. We could always add the file to .gitattributes at the root of our repo.